library/src/main/java/com/schibsted/spain/barista/custom/DisplayedMatchers.java

```diff
@@ -0,0 +1,31 @@
+package com.schibsted.spain.barista.custom;
+
+import android.support.annotation.IdRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.StringRes;
+import android.view.View;
+
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.core.AllOf.allOf;
+
+public class DisplayedMatchers {
```

Comment:
But this code is not part of the public API, isn't it? It's code that we call from Barista. I know that someone can call it cos it's public, but... that's not what Barista aims to do.

In the other hand, the method does exactly what it explains. I mean, if I know how Espresso works, if I see a matcher called `displayedWithId(@ResId int id)`, it sounds that it matches with displayed views that had the given ID.

Let me skip adding the Javadoc. If it's a issue, we will add in as soon as it adds value to Barista users.